apis/scheme/accompany.ts

```diff
@@ -26,3 +26,15 @@ export interface CompanionDetail
   viewCount: number;
   waitingCount: number;
 }
+
+export interface CompanionRecruitmentCard {
+  title: string;
+  concertName: string;
+  userId: string;
+  gender: string;
```

Comment:
```suggestion
  gender: '남' | '여' | '무관';
```
요 필드도, 이렇게 하는것이 나을까요??🤨